### PR TITLE
chore: keep package.json files up to date

### PR DESCRIPTION
**Pull request by ashuralyk**
**What this PR does / why we need it**:

This PR

**Which issue(s) this PR fixes**:

Fixes #

**Which docs this PR relation**:

Ref #

**Which toolchain this PR adaption**:

No Breaking Change

**Special notes for your reviewer**:

NIL

**CI Description**

| CI Name                                   | Description                                                     |
| ----------------------------------------- | --------------------------------------------------------------- |
| *Chaos CI*                                | Test the liveness and robustness of Axon under terrible network condition    |
| *Cargo Clippy*                            | Run `cargo clippy --all --all-targets --all-features`      |
| *Coverage Test*                           | Get the unit test coverage report                             |
| *E2E Test*                                | Run end-to-end test to check interfaces                         |
| *Code Format*                             | Run `cargo +nightly fmt --all -- --check` and `cargo sort -gwc`     |
| *Web3 Compatible Test*                    | Test the Web3 compatibility of Axon                               |
| *v3 Core Test*                            | Run the compatibility tests provided by Uniswap V3             |
| *OCT 1-5 \| 6-10 \| 11 \| 12-15 \| 16-19* | Run the compatibility tests provided by OpenZeppelin           |

**CI Usage**

> Check the CI you want to run below, and then comment `/run-ci`.

**CI Switch**

- [ ] Chaos CI
- [ ] Cargo Clippy
- [ ] Coverage Test
- [x] E2E Tests
- [ ] Code Format
- [x] Unit Tests
- [ ] Web3 Compatible Tests
- [ ] OCT 1-5 And 12-15
- [ ] OCT 6-10
- [ ] OCT 11
- [ ] OCT 16-19
- [ ] v3 Core Tests

### CODE REVIEW

**Review by KaoImin — DISMISSED** [review]

File: tests/e2e/package.json [hunk 1]
```diff
@@ -6,11 +6,11 @@
     "lint": "eslint src/*.js ./*.js"
   },
   "dependencies": {
-    "@chainsafe/dappeteer": "^2.4.0",
+    "@chainsafe/dappeteer": "^3.0.0",
```
Comment (hongda3141) [inline tests/e2e/package.json]: Compatibility of this version may be have bugs, specific details( https://github.com/ChainSafe/dappeteer/issues/155 ). I am considering other ways as backup plan.

File: tests/e2e/package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-sonarjs": "^0.15.0",
     "http-server": "^14.1.1",
-    "jest": "^29.0.1",
+    "jest": "^29.1.2",
```
Comment (hongda3141) [inline tests/e2e/package.json]:
```suggestion
    "jest": "^29.2.0",
```

File: tests/e2e/package.json [hunk 3]
```diff
@@ -6,11 +6,11 @@
...
   },
   "dependencies": {
-    "@chainsafe/dappeteer": "^2.4.0",
+    "@chainsafe/dappeteer": "^3.0.0",
     "ethereumjs-tx": "^2.1.2",
-    "jest-environment-node": "^28.1.3",
-    "puppeteer": "^17.0.0",
-    "web3": "^1.7.5",
+    "jest-environment-node": "^29.1.2",
+    "puppeteer": "^18.0.5",
```
Comment (hongda3141) [inline tests/e2e/package.json]:
```suggestion
    "puppeteer": "^19.0.0",
```

File: tests/e2e/package.json [hunk 4]
```diff
@@ -6,11 +6,11 @@
     "lint": "eslint src/*.js ./*.js"
   },
   "dependencies": {
-    "@chainsafe/dappeteer": "^2.4.0",
+    "@chainsafe/dappeteer": "^3.0.0",
     "ethereumjs-tx": "^2.1.2",
-    "jest-environment-node": "^28.1.3",
-    "puppeteer": "^17.0.0",
-    "web3": "^1.7.5",
+    "jest-environment-node": "^29.1.2",
```
Comment (hongda3141) [inline tests/e2e/package.json]:
```suggestion
```

new version jest make "jest-environment-node" as a buildin package. :smile:

**Review by KaoImin — APPROVED** [review]

**Pull request merged**